connector/src/main/scala/com/google/cloud/spark/bigquery/SparkBigQueryOptions.scala

```diff
@@ -143,12 +148,24 @@ object SparkBigQueryOptions {
     val optimizedEmptyProjection = getAnyBooleanOption(
       allConf, parameters, "optimizedEmptyProjection", true)
 
+    val allowFieldAddition = getAnyBooleanOption(
```

Comment:
do we need unit tests for this? Just to make sure that the option value got updated correctly.

Reply:
Of course, pushed them now.